packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -131,6 +134,14 @@ export namespace WorkspaceCommands {
         category: 'File',
         label: 'Save As...',
     };
+    export const COPY_PATH: Command = {
+        id: 'core.copy.path',
+        label: 'Copy Path'
```

Comment:
Providing a `label` means that this command will be accessible through the command palette.
Is this the intended behavior? Else, I think renaming the command makes more sense.
For example, in VSCode the command is `Copy Path of Active File` with a `File Category`.